{
  "name": "fun-try-catch",
  "version": "1.0.0",
  "description": "A functional try catch for javascript and typescript ",
  "scripts": {
    "build": "tsup --config tsup.config.ts",
    "lint": "eslint ./src",
    "test": "jest --watch",
    "format": "prettier ./src --write"
  },
  "keywords": [
    "try-catch",
    "functional",
    "promise",
    "async",
    "await",
    "typescript",
    "javascript",
    "try",
    "catch",
    "error",
    "exception",
    "error-handling"
  ],
  "author": {
    "name": "Diogo Passos",
    "url": "https://diogopassos.pt"
  },
  "homepage": "https://github.com/gngz/fun-try-catch",
  "repository": "https://github.com/gngz/fun-try-catch",
  "bugs": {
    "url": "https://github.com/gngz/fun-try-catch/issues"
  },
  "license": "MIT",
  "packageManager": "pnpm@10.6.2",
  "files": [
    "dist",
    "package.json"
  ],
  "exports": {
    ".": {
      "types": "./dist/try-catch.d.ts",
      "require": "./dist/try-catch.js",
      "import": "./dist/try-catch.mjs"
    }
  },
  "main": "./dist/try-catch.js",
  "module": "./dist/try-catch.mjs",
  "types": "./dist/try-catch.d.ts",
  "devDependencies": {
    "@eslint/js": "^9.22.0",
    "@jest/globals": "^29.7.0",
    "@types/jest": "^29.5.14",
    "eslint": "^9.22.0",
    "eslint-config-prettier": "^10.1.1",
    "globals": "^16.0.0",
    "jest": "^29.7.0",
    "prettier": "3.5.3",
    "ts-jest": "^29.2.6",
    "tsup": "^8.4.0",
    "typescript": "^5.8.2",
    "typescript-eslint": "^8.26.1"
  }
}